app.js

```diff
@@ -1,88 +1,751 @@
 // Task 1 - Notes for JS Evaluation Predictions
 
 // List 3 expressions that surprised you or gave unexpected results, why?
-
+    /* 
+    2 + 'pizza' = '2pizza' ????
+    5 === 5.0 = true. I thought this would be false because of the decimal point
```

Comment:
In JS, floats (decimals) and integers are seen the same. JS actually stores numbers as floats so even if you input `5` JS sees it as `5.0`